chrome/content/zotero/elements/contextPane.js

```diff
@@ -171,7 +171,7 @@
 				ZoteroContextPane.showLoadingMessage(false);
 				this._sidenav.hidden = true;
 			}
-			else if (Zotero_Tabs.selectedType.includes('reader')) {
+			else if (Zotero_Tabs.selectedType == 'reader') {
```

Comment:
This was presumably breaking something. Do we know what?

Reply:
I initially changed it to be Zotero_Tabs.selectedType.includes('reader') as I was running into issues with context pane sometimes not properly appearing during (granted, quite aggressive) tab switching. At that point, a tab could be either reader or reader-unloaded, so this seemed appropriate. Though, I went back to https://github.com/zotero/zotero/commit/3f45def928633be3dd4f5d8fcbedb6b589b46236 and I couldn't reproduce that issue I was having with just Zotero_Tabs.selectedType == 'reader'. Perhaps the issue I saw came from a VM being slow or something like that and the includes part is really not required.

As far as I can see, the this._handleReaderReady(reader);below would be a noop for a tab whose reader is not yet loaded, and then we just set the contextPane collapsed status and sidenav visibility, so I would expect no serious breakages there.

Reply:
Oh, I didn't realize it was targeting `reader-unloaded`. I think `.startsWith('reader')` would be clearer (and nominally faster) and look less like a type mistake. But if we don't need it, that's fine too.